config/consts.rb

```diff
@@ -7,7 +7,7 @@ class ImageConfigData
 
   def initialize
     root = File.expand_path("..", __dir__)
-    @env = ENV["RAILS_ENV"] || "development"
+    @env = ENV.fetch("RAILS_ENV", "development")
```

Comment:
I think this is the cop in this PR which results in longer code. It's safer this way.
See Style/FetchEnvVar https://docs.rubocop.org/rubocop/1.31/cops_style.html#stylefetchenvvar